src/esphomelib/api/api_server.cpp

```diff
@@ -54,7 +54,7 @@ void APIServer::loop() {
   }
 }
 void APIServer::dump_config() {
-
+  ESP_LOGI(TAG,"API Server:");
```

Comment:
This is not an info message, it's a config message.
Space missing after comma.

Reply:
sure, I will use ESP_LOGCONFIG().Thanks.